flow-typed/interfaces/engine-decorator-provider.js

```diff
@@ -1,6 +0,0 @@
-// @flow
-import type {IEngine} from './engine';
-
-declare interface IEngineDecoratorProvider {
-  getEngineDecorator(engine: IEngine, dispatchEventHandler: Function): IEngineDecorator;
```

Comment:
this needs to be part of the plugin interface, it is ok to remove it from here but need to make sure we define it for plugins in kaltura-player-js

Reply:
Sure the change for kaltura player didn't upload yet